test/config.json

```diff
@@ -1008,6 +1008,15 @@
 			"RetryMax": 2,
 			"Tags": []
 		},
+		"vreplication_vschema_load": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vreplication", "-run", "TestVSchemaChangesUnderLoad"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vreplication_cellalias",
```

Comment:
We should probably rename the shard, e.g. vreplication_cellalias_vschema or something.